src/RamStatement.h

```diff
@@ -774,4 +774,43 @@ class RamLogSize : public RamRelationStatement {
     std::string message;
 };
 
+/**
+ * @class RamCall
+ * @brief Call a subroutine
+ *
+ * Calls a subroutine
+ *
+ * The following example shows how subroutine A is invoked
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ * CALL A
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ */
+
+class RamCall : public RamStatement {
+public:
+    RamCall(std::string name) : name(std::move(name)) {}
+
+    /** @brief Get exit condition */
```

Comment:
The brief doesn't seem appropriate for the method name. Are you sure?

Reply:
Thta is wrong indeed - well spotted.